### ISSUE TITLE
get_primary_monitor() returns None

### ISSUE THREAD

**Issue by Triggaaaaa** [Cimbali/pympress#285]
**Describe the bug**
Upon last update to version 1.8.0 pympress does not start anymore. 

**To Reproduce**
Launch `pympress`.

**Expected behavior**
Normal startup as before.

**Environment**
 - OS: Arch Linux
 - Wayland
 - KDE
 - Python version: 3.10.10
 - Pympress version: 1.8.0
 - Gtk 3.24.37
 - GLib 2.76.1
 - Poppler 23.03.0 cairo
 - Cairo 1.17.8
 - pycairo 1.23.0
 - Installation method: Arch Linux AUR, which builds from source

**Debug information**
 - What is reported in pympress.log?
```
INFO:pympress.app:Pympress: 1.8.0 ; Python: 3.10.10 ; OS: Linux 6.2.8-arch1-1 #1 SMP PREEMPT_DYNAMIC Wed, 22 Mar 2023 22:52:35 +0000 ; Gtk 3.24.37 ; GLib 2.76.1 ; Poppler 23.03.0 cairo ; Cairo 1.17.8 , pycairo 1.23.0
INFO:pympress.extras:Media support: GdkPixbuf gif player, GStreamer 1.22.1, VLC not available
CRITICAL:pympress.__main__:Uncaught exception:
Traceback (most recent call last):
  File "/usr/lib/python3.10/site-packages/pympress/app.py", line 227, in do_open
    self.do_activate(timestamp=GLib.get_current_time())
  File "/usr/lib/python3.10/site-packages/pympress/app.py", line 157, in do_activate
    self.gui = ui.UI(self, self.config)
  File "/usr/lib/python3.10/site-packages/pympress/ui.py", line 317, in __init__
    self.setup_screens(screen, 'at startup')
  File "/usr/lib/python3.10/site-packages/pympress/ui.py", line 484, in setup_screens
    c_monitor, p_monitor, primary, non_primary = ScreenArea.lookup_monitors(display, self.c_win, self.p_win)
  File "/usr/lib/python3.10/site-packages/pympress/ui.py", line 2022, in lookup_monitors
    prim = ScreenArea(display.get_primary_monitor().get_geometry())
AttributeError: 'NoneType' object has no attribute 'get_geometry'
```

 - Does the problem still happen if you remove your config file?
 Yes.

**Additional context**
Changing `get_primary_monitor()` to e.g. `get_monitor(0)` is a workaround.

**Comment by Cimbali** [comment]
Thanks for reporting! This is he same issue as #283, and he fix is already released in v1.8.1.

**Comment by Triggaaaaa** [comment]
Thank you!
Sorry for missing #283
